spec/features/access_cookies_spec.rb

```diff
@@ -0,0 +1,369 @@
+require 'rails_helper'
+
+RSpec.feature "Access Cookies", js: true do
+  include ET3::Test::I18n
+
+  before do
+    stub_et_api
+    stub_build_blob_to_azure
+  end
+
+  scenario "from start page" do
+    start_page.load(locale: current_locale_parameter)
+
+    start_page.cookies.click
+
+    expect(cookies_page).to be_displayed
```

Comment:
This looks like a perfect example of where 'shared examples' would read better and DRY this up without it being stored in separate files - i.e. define the shared examples at the top, then setup a context for each variation with a let block for the 'start_page' (which looks like the only thing that varies).

Or, another option is make the page object work harder - so have a single method that checks everything ?

Reply:
Great idea - amended in 1b39207.